chapter1/poisson.ipynb

```diff
@@ -55,12 +55,16 @@
     "from sympde.topology import ScalarFunctionSpace, Square, element_of\n",
     "from sympde.calculus import grad, dot\n",
     "\n",
-    "from sympy import pi, sin\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from mpl_toolkits.axes_grid1 import make_axes_locatable\n",
     "\n",
-    "domain = Square()\n",
+    "from sympy import pi, sin, lambdify\n",
```

Comment:
git sees some changes because this import now comes after the line break. If you keep the same line breaks than in the original code, git will only see the relevant changes and the commit will be a bit cleaner.